rholang/src/test/scala/coop/rchain/rholang/interpreter/CostAccountingReducerTest.scala

```diff
@@ -110,9 +110,15 @@ class CostAccountingReducerTest extends FlatSpec with Matchers with TripleEquals
       )
     ] = {
 
+      implicit val cost: _cost[Task] =
+        (for {
+          costAlg <- CostAccounting.of[Task](Cost(0))
+        } yield loggingCost(costAlg, noOpCostLog[Task]))
+          .runSyncUnsafe(1.second)
```

Comment:
is 1 second a good rule of thumb or is it just coincidental?

Reply:
Consistency. This is what's being done in the test code in that area